acct/check

```diff
@@ -132,11 +135,31 @@ if [ -f "$vhost_file" ] && vhost=$(grep -E "^[#[:space:]]*$user[![:space:]]" "$v
 fi
 
 # check apphosting config
-if [ -f "$vhost_app_file" ] && vhost=$(grep -E "^[#[:space:]]*$user[![:space:]]" "$vhost_app_file"); then
+if [ -f "$vhost_app_file" ] && vhost_app=$(grep -E "^[#[:space:]]*$user[![:space:]]" "$vhost_app_file"); then
   echo
   echo "Virtual apphosts:"
   echo "-------------------------------------------------------------------------------"
-  echo "$vhost"
+  echo "$vhost_app"
+  echo "-------------------------------------------------------------------------------"
+fi
+
+# check virtual mail config
+if [ -f "$vhost_mail_file" ] && vhost_mail=$(grep -E "^[#[:space:]]*$user[![:space:]]" "$vhost_mail_file"); then
+  echo
+  echo "Virtual mail:"
+  echo "-------------------------------------------------------------------------------"
+  echo "$vhost_mail"
+  echo "-------------------------------------------------------------------------------"
+fi
+
+# check dns entries
+dns=$(./check-dns "$user")
```

Comment:
I think you'll want something like `"$(dirname "$(readlink -f -- "$0")")"/check-dns` here instead

Reply:
what does this do differently besides call the script directly instead of through the symlink? I'm not sure I understand the difference?

Reply:
Well, the way you have it now, it won't work after you deploy it unless your current directory happens to be `/opt/share/utils/bin`

Reply:
oh, that makes sense. thanks